packages/core/src/com/communication.ts

```diff
@@ -765,7 +767,7 @@ export class Communication {
         }
     }
 
-    public handleReady({ from }: ReadyMessage): void {
+    public handleReady({ from }: { from: string }): void {
```

Comment:
What flow would you like to call this manually? 
You can also remove all the `as ReadyMessage` from the other places that call this

Reply:
manually state the env is ready or pending. (like the places you saw the cast at)
I removed all type casts